Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml

```
@@ -554,6 +642,14 @@
   components:
     - type: Sprite
       state: poster58_contraband
+    - type: ExtendDescription
+      descriptionList:
+        - description: "mindshield-extenddescription-poster-cybersun600"
+          fontSize: 12
+          color: "#11aaff"
+          requireDetailRange: false
+          requirements:
+          - !type:CharacterMindshieldRequirement
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 12 but found 10

(indentation)